### ISSUE TITLE
Indicator freezes wingpanel

### ISSUE THREAD

**Issue by elementaryBot** [elementary/wingpanel-indicator-power#10]
I found a number of users with this same bug on Stack Exchange: http://elementaryos.stackexchange.com/questions/7680/wingpanel-not-responding-loki (unlike the original poster, I did not install Dropbox—only major install was the Google Chrome repo and app)

I returned from system sleep to find the menu bar gone. When I Ctrl+Alt+Del, I get a notification that wingpanel is not responding, but no feedback if I opt for force quitting it. New Loki install, Intel integrated graphics on Gigabyte GA-H110M-A motherboard.

"sudo killall wingpanel" has no immediate change, but shows a small, white "wiping"-style movement in the top right, like when a notification closes. Sometimes the menu bar restores, but none of the icons or Applications menu respond. (Once I saw the bottom of the message bubble-style popout come out of "Applications", with the triangle part of the bubble pointing to Applications, but all but the bottom white margin of the bubble off the top of the screen. Eventually, I get another notice about how wingpanel is not responding. Force quitting starts the same process anew. I'm unable to get a working menu bar even after rebooting.

Update: The menu bar does become functional again, but only after a notification that I am connected to wired ethernet (even though I haven't touched the computer or installed anything to prompt the change). It quickly becomes unresponsive again.

Launchpad Details: [#LP1623793](https://bugs.launchpad.net/bugs/1623793) wolf - 2016-09-15 06:04:52 +0000

**Comment by elementaryBot** [comment]
Hey, unfortunatly I can't reproduce this.
If you or somebody have time to test some stuff it would be great. 
It would be helpful to know if it is a indicator or wingpanel.
You could remove all indicators with:
sudo apt-get remove wingpanel-indicator-ayatana wingpanel-indicator-session wingpanel-indicator-bluetooth wingpanel-indicator-network wingpanel-indicator-sound wingpanel-indicator-datetime wingpanel-indicator-notifications wingpanel-indicator-keyboard wingpanel-indicator-power 

and test if it is better. If yes add one indicator at a time and figure out which is it.

To get all back just run:
sudo apt-get install wingpanel-indicator-ayatana wingpanel-indicator-session wingpanel-indicator-bluetooth wingpanel-indicator-network wingpanel-indicator-sound wingpanel-indicator-datetime wingpanel-indicator-notifications wingpanel-indicator-keyboard wingpanel-indicator-power 

Launchpad Details: [#LPC](https://bugs.launchpad.net/wingpanel-indicator-power/+bug/1623793/comments/1) Djax - 2016-09-15 07:47:28 +0000

**Comment by elementaryBot** [comment]
✓ wingpanel-indicator-ayatana
✓ wingpanel-indicator-session
✓ wingpanel-indicator-bluetooth
✓ wingpanel-indicator-network
✓ wingpanel-indicator-sound
✓ wingpanel-indicator-datetime
✓ wingpanel-indicator-notifications
✓ wingpanel-indicator-keyboard
✗ wingpanel-indicator-power

Looks like it's wingpanel-indicator-power. It crashes wingpanel each time I install it, no matter what else is installed. Once I uninstall it and restart wingpanel, it appears to work fine.

Launchpad Details: [#LPC](https://bugs.launchpad.net/wingpanel-indicator-power/+bug/1623793/comments/2) wolf - 2016-09-15 19:07:32 +0000

**Comment by elementaryBot** [comment]
Same here, without wingpanel-indicator-power everything works. After installing it the panel does not start correctly.

Launchpad Details: [#LPC](https://bugs.launchpad.net/wingpanel-indicator-power/+bug/1623793/comments/3) Tobias - 2016-09-16 06:24:03 +0000

**Comment by elementaryBot** [comment]
Thank you wolf. This is really helpful to near it down to the right indicator.

Launchpad Details: [#LPC](https://bugs.launchpad.net/wingpanel-indicator-power/+bug/1623793/comments/4) Djax - 2016-09-16 07:47:51 +0000

**Comment by elementaryBot** [comment]
I can confirm wing panel bug. But in my case it is 'wingpanel-indicator-session' package which causes the crash. With rest of the package all works fine. I am happy to report further logs if you guys guide me. 

Launchpad Details: [#LPC](https://bugs.launchpad.net/wingpanel-indicator-power/+bug/1623793/comments/5) Sundar - 2016-09-21 16:16:40 +0000

**Comment by elementaryBot** [comment]
I have solved my issue and you can look at what I have done here https://bugs.launchpad.net/elementaryos/+bug/1596377/comments/14

Launchpad Details: [#LPC](https://bugs.launchpad.net/wingpanel-indicator-power/+bug/1623793/comments/6) Sundar - 2016-09-22 16:56:20 +0000

**Comment by elementaryBot** [comment]
I just noticed on a fresh Loki installation that this bug only affects me if I use an external monitor. Without it I can reinstall and use `wingpanel-indicator-power` without any problems.
I'm running on a Dell XPS 13, but had the same problem on a Samsung Series 9 device.

Launchpad Details: [#LPC](https://bugs.launchpad.net/wingpanel-indicator-power/+bug/1623793/comments/7) Tobias - 2017-01-07 08:14:39 +0000

**Comment by elementaryBot** [comment]
I've just discovered https://bugs.launchpad.net/wingpanel-indicator-power/+bug/1650771

I can confirm that the wingpanel "unfreezes" as soon as I unplug my Logitech receiver. It seems to be the cause of the problem in my case (I use the receiver only while connected to an external screen and have used it on both my Samsung series 9 and Dell XPS 13)

Launchpad Details: [#LPC](https://bugs.launchpad.net/wingpanel-indicator-power/+bug/1623793/comments/8) Tobias - 2017-01-07 08:35:26 +0000

**Comment by danrabbit** [comment]
Closing since this seems to be a duplicate of #15
